src/adapters/gitlab.js

```diff
@@ -89,6 +91,8 @@ class GitLab extends Adapter {
       $(GL_HEADER).css('z-index', 3)
       $(GL_SIDEBAR).css('z-index', 1)
       $(GL_TITLE).css('margin-left',  sidebarVisible ? '' : 56)
+      $(GL_CONTENT).parent().css('margin-left', sidebarVisible ? 230 : '')
```

Comment:
I have changed to fix display unfriendly by adding styles as the new commit.
If finally we decide to use this method, I will refactor the code as below.
```
// Special for GitLab 7.5.3 b656b85
$(GL_CONTENT).parent().add(GL_MAIN_NAV).css('margin-left', sidebarVisible ? 230 : '')
```